frappe/templates/includes/comments/comments.py

```diff
@@ -29,7 +30,7 @@ def add_comment(comment, comment_email, comment_by, reference_doctype, reference
 		return False
 
 	comment = doc.add_comment(
-		text=comment,
+		text=clean_html(comment),
```

Comment:
why not do this in `add_comment` in `document.py`?

Reply:
I ignored it, since it was harmless anyway in the doctype. Any script was not triggered.
![Screenshot 2022-02-04 at 11 51 06 AM](https://user-images.githubusercontent.com/30501401/152482183-8cccbd73-cd24-435a-85ac-bd58497bbcfb.png)